h/templates/activity/search.html.jinja2

```diff
@@ -348,18 +348,26 @@
     {% endif %}
 
     {% if timeframes %}
-      <ol class="search-result-list {%- if more_info %} search-result-hide-on-small-screens{% endif %}">
-        {% for timeframe in timeframes %}
+
+      <div class="search-results">
```

Comment:
Hmm, I wonder if there is a better name for this class to distinguish it from `.search-result-container`. If that is hard to come by, it would be helpful to add some comments in `search.scss` to clarify the difference between the two.